src/components/Pages/RequestPage/RequestPage.tsx

```diff
@@ -179,24 +202,69 @@ export const RequestPage = () => {
           case 'dcl_personal_sign':
             setView(View.VERIFY_SIGN_IN)
             break
-          case 'eth_sendTransaction':
+          case 'eth_sendTransaction': {
+            // Get wallet info first
+            const signer = await browserProvider.current.getSigner()
+            const userBalance = await browserProvider.current.getBalance(signer.address)
+            const currentChainId = await browserProvider.current.getNetwork().then(network => Number(network.chainId))
+            setWalletInfo({
+              balance: userBalance,
+              chainId: currentChainId
+            })
```

Comment:
All of this could still be done under the try below as it was done before, taking into consideration that it will only be necessary to get the gas price of the transaction if we don't need to perform a meta transaction. WDYT about moving it to it's original palce?